From 82ae63e2b85073230cc3b6a76499448d01f966ca Mon Sep 17 00:00:00 2001
Message-Id: <82ae63e2b85073230cc3b6a76499448d01f966ca.1585650704.git.pmatilai@redhat.com>
From: Panu Matilainen <pmatilai@redhat.com>
Date: Tue, 31 Mar 2020 13:29:48 +0300
Subject: [PATCH] Unset SOURCE_DATE_EPOCH for the test-suite

Fixes the reproducable build test failing in Fedora rpm builds due to
%source_date_epoch_from_changelog being set on the outside, which
leaks the SOURCE_DATE_EPOCH environment into the test-suite and
changes the expectation.
---
 tests/atlocal.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/atlocal.in b/tests/atlocal.in
index 95baca55f..a41752f5e 100644
--- a/tests/atlocal.in
+++ b/tests/atlocal.in
@@ -27,6 +27,8 @@ export HOME
 TZ=UTC
 export TZ
 
+unset SOURCE_DATE_EPOCH
+
 TOPDIR="${RPMTEST}/build"
 
 RPM_XFAIL=${RPM_XFAIL-1}
-- 
2.25.1

